### ISSUE TITLE
 Template creation flow voiceover gets stuck in a loop

### ISSUE THREAD

**Issue by heyitsmebev** [GSA/notifications-admin#2056]
-Template creation flow voiceover gets stuck in a loop
1. When tabbing through different elements within the template creation flow, I get stuck in a loop where I cannot check  radio buttons (start with blank page or copy existing page).
2. When tabbing through different element, I cannot check any of the existing templates boxes
3. Continue button also does not work with voiceover 
4. Cancel button does work within the template creation option

![Image](https://github.com/user-attachments/assets/4d0841aa-ca5f-4ac5-9051-2e14fc42b681)

_Originally posted by @heyitsmebev in https://github.com/GSA/notifications-admin/issues/1940#issuecomment-2429853746_

**Comment by heyitsmebev** [comment]
I initially reported an issue with selecting checkboxes and radio buttons. However, I learned that I need to press the space bar to select and the arrow keys to move through different options. I no longer get stuck in a loop.
